pallets/anchors/src/lib.rs

```diff
@@ -89,6 +90,7 @@ pub mod pallet {
 	// method.
 	#[pallet::pallet]
 	#[pallet::generate_store(pub (super) trait Store)]
+	#[pallet::without_storage_info]
```

Comment:
What is this for?

Reply:
Looks related to https://github.com/paritytech/substrate/pull/10662.
Sounds like this attribute is a bit of a hack and we should work on being able to remove it? But that can be a future upgrade, the less we change when we're in a hurry the better.

Would be nice to ticket this, though.